### DISCUSSION TITLE
Cannot log in. Error 403 forbidden

### DISCUSSION THREAD

**Discussion by Miles-and-miles-away** [dandi/helpdesk#90] [Q&A]
Tried logging in to Dandi hub using either firefox or safari. Both result in "Forbidden" error.

**Comment by satra** [comment]
One cannot log into dandi hub without a registration for dandiarchive.org being approved. did you go to https://dandiarchive.org to register first?

**Comment by Miles-and-miles-away** [comment]
Thanks. I just registered. Hopefully they’ll approve soon


From: Satrajit Ghosh ***@***.***>
Date: Thursday, 9March, 2023 8:47
To: dandi/helpdesk ***@***.***>
Cc: Miles Desforges ***@***.***>, Author ***@***.***>
Subject: Re: [dandi/helpdesk] Cannot log in. Error 403 forbidden (Discussion #90)

One cannot log into dandi hub without a registration for dandiarchive.org being approved. did you go to https://dandiarchive.org to register first?
